.github/workflows/flake-updates-nixpkgs-update.yml

```diff
@@ -13,12 +13,8 @@ jobs:
           extra_nix_config: |
             experimental-features = nix-command flakes
       - name: Update flake.lock
-        id: update
         uses: DeterminateSystems/update-flake-lock@v19
         with:
-          token: ${{ secrets.GH_TOKEN_FOR_UPDATES }}
```

Comment:
I think you still need the following permissions:

```
permissions:
  pull-requests: write
  contents: write
```

Reply:
In this context the action has write so we don't need to set permissions unless we want to restrict write to specific steps.

Reply:
Ok. I only remember this being not the case in some orgs also I haven't check when this is the case.